releases/protocol/release-specs-v1.6.0.md

```diff
@@ -0,0 +1,712 @@
+[comment]: <> (last PR processed #5490)
+
+# Contents
+
+This document explains the contents of the rc/v1.6.0 release. It is split in 3 sections: 
```

Comment:
might refer to `rc/v1.6.0` as Sirius as well (inside these files, the PR title, description and so on)

Reply:
done